src/Sentry/Contexts.cs

```diff
@@ -111,8 +111,11 @@ internal void CopyTo(Contexts to)
     }
 
     /// <inheritdoc />
-    public void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger? logger) =>
-        writer.WriteDictionaryValue(this!, logger, includeNullValues: false);
+    public void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger? logger)
+    {
+        var contexts = this.OrderBy(x => x.Key, StringComparer.Ordinal);
```

Comment:
I assume the cost associated with sorting it all is negligent?

Reply:
Verify test became flaky after updating, but this was the root cause.  Verify appearing to be be flaky just highlighted that we weren't getting consistent sort order here in the first place.

Cost is negligent - yes.  Contexts are pretty small anyway.